cki_lib/libcki.sh

```diff
@@ -8,7 +8,30 @@
 #       2) All global variables start with 'CKI_'.
 #
 
-source /usr/bin/rhts_environment.sh
+# Set CKI test environment
+if [ -z "$OUTPUTFILE" ]; then
+        export OUTPUTFILE=`mktemp /mnt/testarea/tmp.XXXXXX`
```

Comment:
the background is:
since rhts environment  script for initial setting test environment, is deprecated in restraint. 
rhts-enviromnet got copied to lib_cki without modification

is identical with rhts-envirnment.sh from restraint-rhts.rpm (package)